packages/neutrino-preset-web/index.js

```diff
@@ -99,6 +99,10 @@ module.exports = (neutrino) => {
       .chunkFilename('[id].[chunkhash].js')
       .end()
     .resolve
+    .alias
+        // TIP: make sure 2 version of "core-js" always match in package.json and babel-polyfill/package.json
```

Comment:
- We can remove the "TIP" word here
- Capitalize "make" to "Make"
- Replace "version" with "versions"